### PR TITLE
translated the missing elements as I thought normal.

### PR DESCRIPTION
**Pull request by ganqqwerty**
Pardon my Italian. Or German. Well, I'm not that good in French either.

### CODE REVIEW

File: alv-portal-ui/translations.csv [hunk 1]
```diff
@@ -1250,18 +1250,18 @@ portal.competence-catalog.ch-fiches.removed-competence-set-success-notification,
...
+portal.competence-catalog.ch-fiches.edit-description-modal.description-title-en,Englisch Beschreibung,English description,Description en anglais,Descrizione inglese,
+portal.competence-catalog.ch-fiches.edit-description-success-notification,CH Fiche Beschreibung gespeichert,CH Fiche description has been saved,La description de CH Fiche a été enregistrée.,La CH Fiche descrizione è stata salvata.,
+portal.competence-catalog.ch-fiches.remove-description-success-notification,CH Fiche Beschreibung gelösht,CH Fiche description has been deleted,La description de CH Fiche a été supprimée.,La CH Fiche descrizione è stata cancellata.,
+portal.competence-catalog.ch-fiches.remove-description-dialog.title,Beschreibung löschen?,Delete description,Supprimer la description?,Descrizione cancellata?,
+portal.competence-catalog.ch-fiches.remove-description-dialog.confirmation-question,Möchten Sie die beschreibung löschen?,Do you really want to delete the description,Voulez-vous vraiment supprimer la description de CH Fiche?,Volete cambiare davvero La CH Fiche descrizione cancellata?,
```
Comment (dvoracek-martin) [inline alv-portal-ui/translations.csv]: You're missing a question mark in the English translation

File: alv-portal-ui/translations.csv [hunk 2]
```diff
@@ -1250,18 +1250,18 @@ portal.competence-catalog.ch-fiches.removed-competence-set-success-notification,
...
+portal.competence-catalog.ch-fiches.edit-description-modal.description-title-fr,Französisch Beschreibung,French description,Description en français,Descrizione francese,
+portal.competence-catalog.ch-fiches.edit-description-modal.description-title-it,Italienisch Beschreibung,Italian description,Description italienne,Descrizione italiana,
+portal.competence-catalog.ch-fiches.edit-description-modal.description-title-en,Englisch Beschreibung,English description,Description en anglais,Descrizione inglese,
+portal.competence-catalog.ch-fiches.edit-description-success-notification,CH Fiche Beschreibung gespeichert,CH Fiche description has been saved,La description de CH Fiche a été enregistrée.,La CH Fiche descrizione è stata salvata.,
+portal.competence-catalog.ch-fiches.remove-description-success-notification,CH Fiche Beschreibung gelösht,CH Fiche description has been deleted,La description de CH Fiche a été supprimée.,La CH Fiche descrizione è stata cancellata.,
+portal.competence-catalog.ch-fiches.remove-description-dialog.title,Beschreibung löschen?,Delete description,Supprimer la description?,Descrizione cancellata?,
```
Comment (dvoracek-martin) [inline alv-portal-ui/translations.csv]: A missing question mark in English

File: alv-portal-ui/translations.csv [hunk 3]
```diff
@@ -1250,18 +1250,18 @@ portal.competence-catalog.ch-fiches.removed-competence-set-success-notification,
...
-portal.competence-catalog.ch-fiches.remove-description-success-notification,CH Fiche Beschreibung gelösht,CH Fiche description has been deleted,,,
-portal.competence-catalog.ch-fiches.remove-description-dialog.title,Beschreibung löschen?,,,,
-portal.competence-catalog.ch-fiches.remove-description-dialog.confirmation-question,Beschreibung löschen?,,,,
-portal.competence-catalog.ch-fiches.remove-description-dialog.confirm,"Ja, löschen",,,,
+portal.competence-catalog.ch-fiches.no-description-placeholder,Keine Beschreibung,No description,Pas de description,Nessuna descrizione,
+portal.competence-catalog.ch-fiches.view-description-modal.title,CH Fiche Beschreibung,CH Fiche description,CH Fiche description,CH Fiche descrizione,
+portal.competence-catalog.ch-fiches.edit-description-modal.title,CH Fiche Beschreibung bearbeiten,Editing CH Fiche description,Modifier la CH Fiche description,Modificare la CH Fiche descrizione,
+portal.competence-catalog.ch-fiches.edit-description-modal.description-title-de,Deutch Beschreibung,German description,Description allemande,Descrizione tedesca,
```
Comment (dvoracek-martin) [inline alv-portal-ui/translations.csv]: Beschreibung is feminine. Hence it should be "Deutsch**e** Beschreibung" . Analogically for French, Italian and English.

File: alv-portal-ui/translations.csv [hunk 4]
```diff
@@ -1250,18 +1250,18 @@ portal.competence-catalog.ch-fiches.removed-competence-set-success-notification,
...
-portal.competence-catalog.ch-fiches.remove-description-dialog.title,Beschreibung löschen?,,,,
-portal.competence-catalog.ch-fiches.remove-description-dialog.confirmation-question,Beschreibung löschen?,,,,
-portal.competence-catalog.ch-fiches.remove-description-dialog.confirm,"Ja, löschen",,,,
+portal.competence-catalog.ch-fiches.no-description-placeholder,Keine Beschreibung,No description,Pas de description,Nessuna descrizione,
+portal.competence-catalog.ch-fiches.view-description-modal.title,CH Fiche Beschreibung,CH Fiche description,CH Fiche description,CH Fiche descrizione,
+portal.competence-catalog.ch-fiches.edit-description-modal.title,CH Fiche Beschreibung bearbeiten,Editing CH Fiche description,Modifier la CH Fiche description,Modificare la CH Fiche descrizione,
+portal.competence-catalog.ch-fiches.edit-description-modal.description-title-de,Deutche Beschreibung,German description,Description allemande,Descrizione tedesca,
+portal.competence-catalog.ch-fiches.edit-description-modal.description-title-fr,Französisch Beschreibung,French description,Description en français,Descrizione francese,
```
Comment (dvoracek-martin) [inline alv-portal-ui/translations.csv]: You corrected the gender in  Deutch Beschreibung -> Deutche Beschreibung. However, you need to do it analogically also for French and Italian and English.  Hence Französisch**e** , Italienisch**e** and Englisch**e**

File: alv-portal-ui/translations.csv [hunk 5]
```diff
@@ -1250,18 +1250,18 @@ portal.competence-catalog.ch-fiches.removed-competence-set-success-notification,
...
-portal.competence-catalog.ch-fiches.remove-description-success-notification,CH Fiche Beschreibung gelösht,CH Fiche description has been deleted,,,
-portal.competence-catalog.ch-fiches.remove-description-dialog.title,Beschreibung löschen?,,,,
-portal.competence-catalog.ch-fiches.remove-description-dialog.confirmation-question,Beschreibung löschen?,,,,
-portal.competence-catalog.ch-fiches.remove-description-dialog.confirm,"Ja, löschen",,,,
+portal.competence-catalog.ch-fiches.no-description-placeholder,Keine Beschreibung,No description,Pas de description,Nessuna descrizione,
+portal.competence-catalog.ch-fiches.view-description-modal.title,CH Fiche Beschreibung,CH Fiche description,CH Fiche description,CH Fiche descrizione,
+portal.competence-catalog.ch-fiches.edit-description-modal.title,CH Fiche Beschreibung bearbeiten,Editing CH Fiche description,Modifier la CH Fiche description,Modificare la CH Fiche descrizione,
+portal.competence-catalog.ch-fiches.edit-description-modal.description-title-de,Deutche Beschreibung,German description,Description allemande,Descrizione tedesca,
```
Comment (cjpsycos) [inline alv-portal-ui/translations.csv]: Description en allemand,Descrizione in tedesco

File: alv-portal-ui/translations.csv [hunk 6]
```diff
@@ -1250,18 +1250,18 @@ portal.competence-catalog.ch-fiches.removed-competence-set-success-notification,
...
+portal.competence-catalog.ch-fiches.edit-description-modal.description-title-fr,Französische Beschreibung,French description,Description en français,Descrizione francese,
```
Comment (cjpsycos) [inline alv-portal-ui/translations.csv]: Description en français,Descrizione in francese

File: alv-portal-ui/translations.csv [hunk 7]
```diff
@@ -1250,18 +1250,18 @@ portal.competence-catalog.ch-fiches.removed-competence-set-success-notification,
...
+portal.competence-catalog.ch-fiches.edit-description-modal.description-title-it,Italienische Beschreibung,Italian description,Description italienne,Descrizione italiana,
```
Comment (cjpsycos) [inline alv-portal-ui/translations.csv]: Description en italien,Descrizione in italiano

File: alv-portal-ui/translations.csv [hunk 8]
```diff
@@ -1250,18 +1250,18 @@ portal.competence-catalog.ch-fiches.removed-competence-set-success-notification,
...
+portal.competence-catalog.ch-fiches.edit-description-modal.description-title-en,Englische Beschreibung,English description,Description en anglais,Descrizione inglese,
```
Comment (cjpsycos) [inline alv-portal-ui/translations.csv]: Description en anglais,Descrizione in inglese

File: alv-portal-ui/translations.csv [hunk 9]
```diff
@@ -1250,18 +1250,18 @@ portal.competence-catalog.ch-fiches.removed-competence-set-success-notification,
...
+portal.competence-catalog.ch-fiches.edit-description-success-notification,CH Fiche Beschreibung gespeichert,CH Fiche description has been saved,La description de CH Fiche a été enregistrée.,La CH Fiche descrizione è stata salvata.,
+portal.competence-catalog.ch-fiches.remove-description-success-notification,CH Fiche Beschreibung gelösht,CH Fiche description has been deleted,La description de CH Fiche a été supprimée.,La CH Fiche descrizione è stata cancellata.,
+portal.competence-catalog.ch-fiches.remove-description-dialog.title,Beschreibung löschen?,Delete description?,Supprimer la description?,Descrizione cancellata?,
+portal.competence-catalog.ch-fiches.remove-description-dialog.confirmation-question,Möchten Sie die beschreibung löschen?,Do you really want to delete the description?,Voulez-vous vraiment supprimer la description de CH Fiche?,Volete cambiare davvero La CH Fiche descrizione cancellata?,
+portal.competence-catalog.ch-fiches.remove-description-dialog.confirm,"Ja, löschen","Yes, delete","Oui, supprimer","Si, cancellare",
```
Comment (cjpsycos) [inline alv-portal-ui/translations.csv]: entity.action.yes-delete

**Pull request merged**